deployments/datastorage/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-#  * Copyright 2020 Intel Corporation.
```

Comment:
Are you sure you need to change the year of issue?

Reply:
I pasted the orignal [code](https://github.com/edgexfoundry/edgex-compose/blob/main/docker-compose-no-secty.yml#L1).

Reply:
Ok, Great!